scripts/eval/yamls/tasks.yaml

```diff
@@ -0,0 +1,69 @@
+icl_tasks:
+-
+  label: jeopardy
+  dataset_uri: eval/local_data/jeopardy_all.jsonl # ADD YOUR OWN DATASET URI
+  num_fewshot: [0, 1, 5, 10]
+  icl_task_type: language_modeling
+  continuation_delimiter: "\nAnswer: " # this separates questions from answers
```

Comment:
@bmosaicml could you spot check all the continuation delimiter choices here? The default is "\n" but then some of these tasks seem to use different ones like `Answer: ` or `\nAnswer: `

Reply:
i believe the default continuation delimiter is space, not sure how `Answer:` vs `\nAnswer` was decided

Reply:
I just double checked them all they're looking correct. Some datasets have newlines appended to the query so there's no need to include it in the Answer delimiter